src/styles/card.scss

```diff
@@ -0,0 +1,75 @@
+.grid-container {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(200px, 200px));
```

Comment:
The `grid-template-columns` property is currently set to `repeat(auto-fit, minmax(200px, 200px))`, which does not meet the task requirement of displaying 1, 2, 3, or 4 columns based on screen resolution. Adjust the grid settings to meet these requirements: 1 column for smaller screens, 2 starting at 488px, 3 starting from 768px, and 4 starting from 1024px.